src/Sulu/Bundle/TestBundle/Resources/app/config/parameters.yml.dist

```diff
@@ -9,6 +9,7 @@ parameters:
     phpcr.password: '%env(PHPCR_PASSWORD)%'
     phpcr.workspace: '%env(PHPCR_WORKSPACE)%'
 
+    default_locale: en
```

Comment:
No this should not be come from here. I see now what is wrong is that the category should use the default_locale from the category entity.

See as example the categories.xml join: https://github.com/sulu/sulu/blob/027b90c68353cebe6e267415ad1e0e11d9dee117/src/Sulu/Bundle/CategoryBundle/Resources/config/lists/categories.xml#L16

So replace the `%default_locale%` with getting the `.default_locale` property from the main category entity.

```suggestion
```